db_store.py

```
@@ -412,7 +412,7 @@ def defrag_availability(db_conn=None):
             # Simple iterative merging logic:
             # 1. Select all blocks sorted by id and start_time
             c.execute(
-                f"SELECT id, {id_col}, start_time, end_time FROM {table} ORDER BY {id_col}, start_time"
+                f"SELECT id, {id_col}, start_time, end_time FROM {table} ORDER BY {id_col}, start_time"  # nosec B608
```

Ruff (0.15.2)

[error] Possible SQL injection vector through string-based query construction

(S608)